### PR TITLE
repository-springboot module

### PR DESCRIPTION
**Pull request by tsurdilo**
adds repository-springboot module which builds the springboot app to run the repository

### CODE REVIEW

File: repository-springboot/README.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+cd repository-springboot
+```
+
+Next you can start the repository with:
+
+```
+java -jar target/repository-springboot-7.7.0-SNAPSHOT.jar
+```
+
+This will boot the repository and connect to controller (by default located at http://localhost:8090/repository - it's
```
Comment (mswiderski) [inline repository-springboot/README.md]: what do you mean by controller here?

**Comment by tsurdilo** [comment]
@mswiderski updated - thanks!

**Comment by tsurdilo** [comment]
Jenkins retest please

**Comment by tsurdilo** [comment]
@mswiderski is ok now? thanks!

**Pull request merged**